bot/kodiak/evaluation.py

```diff
@@ -780,6 +781,10 @@ async def set_status(msg: str, markdown_content: Optional[str] = None) -> None:
                 )
                 return
 
+        if pull_request.reviewDecision == PullRequestReviewDecision.REVIEW_REQUIRED:
```

Comment:
Should this also check for `CHANGES_REQUESTED`? Or maybe `if pull_request.reviewDecision and pull_request.reviewDecision != PullRequestReviewDecision.APPROVED`?

Reply:
I think we're covered by the logic that's just above this new code: https://github.com/chdsbd/kodiak/blob/20792c4ccbd22cfe42120203a68d0e1f230b2855/bot/kodiak/evaluation.py#L755-L784

Although, I think we may be able to replace all of that with pull_request.reviewDecision, in which case well need to handle all of the PullRequestReviewDecision states.